apps/aecore/src/aec_consensus_hc.erl

```diff
@@ -472,6 +472,13 @@ parent_generation() ->
     Fun = fun() -> get_consensus_config_key([<<"parent_chain">>, <<"parent_generation">>]) end,
     aeu_ets_cache:get(?ETS_CACHE_TABLE, parent_generation, Fun).
 
+acceptable_sync_offset() ->
+    Fun = fun() -> get_consensus_config_key([<<"parent_chain">>, <<"acceptable_sync_offset">>], 60000) end,
```

Comment:
Should this not be in percent of a parent epoch or so?

Feels strange that a note can on itself just configure a number. Surely at the end one wants to vote whether or not to increase/decrease child epoch length. Bit on what basis? If each node can have its own config setting and decides based on that, then "consensus" of change is a weird thing.